README.md

```diff
@@ -1562,6 +1562,7 @@ See also [Are we web yet?](https://www.arewewebyet.org) and [Rust web framework
 * Reverse Proxy
   * [sozu-proxy/sozu](https://github.com/sozu-proxy/sozu) [[sozu](https://crates.io/crates/sozu)] — A HTTP reverse proxy. [![CI](https://github.com/sozu-proxy/sozu/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/sozu-proxy/sozu/actions/workflows/ci.yml)
 * Static Site Generators
+  * [grego/blades](https://github.com/grego/blades) [[blades](https://getblades.org/)] — Blazing fast dead simple static site generator.
```

Comment:
This list has to be ordered alphabetically, so please move this entry below (maybe above "leven")

Reply:
Fixed.